frame/bags-list/src/mock.rs

```diff
@@ -101,12 +101,13 @@ pub(crate) const GENESIS_IDS: [(AccountId, VoteWeight); 4] =
 	[(1, 10), (2, 1_000), (3, 1_000), (4, 1_000)];
 
 #[derive(Default)]
-pub(crate) struct ExtBuilder {
+pub struct ExtBuilder {
 	ids: Vec<(AccountId, VoteWeight)>,
 }
 
 impl ExtBuilder {
 	/// Add some AccountIds to insert into `List`.
+	#[cfg(not(feature = "fuzz"))]
```

Comment:
hmm, why?

Reply:
It was an attempt to get the CI not break when it sees unused code .. but seems to be causing other issues in the CI now